example/lib/example_peripheral.dart

```diff
@@ -5,6 +5,19 @@ import 'dart:typed_data';
 import 'package:flutter/foundation.dart';
 import 'package:blemulator/blemulator.dart';
 
+class GenericPeripheral extends SimulatedPeripheral {
```

Comment:
I'd rather you created a directory `example_peripherals` and `example_peripherals.dart` that would export all the peripherals and have both of the types live in separate files.
GenericPeripheral is currently almost the same as SimulatedPeripheral - perhaps it should have some service with some characteristic?

Reply:
Moving to a separate file seems reasonable. 

Currently, there's no need for GenericPeripheral to have services or characteristics defined, but it was not possible to use just SimulatedPeripheral, because it's defined as an abstract class.

Reply:
I understand there's no need, but there's also no need for an example peripheral that is not a peripheral, since it does not accept or expose any data. ;)

Reply:
Allright, I can see the merit to add at least one service and characteristic for consistency and better readability :)